README.md

```diff
@@ -172,6 +172,11 @@ nvim_lsp.SERVER.setup{config}
   The `config` parameter has the same shape as that of
   |vim.lsp.start_client()|, with these additions and changes:
 
+  {cmd}
+    Required for all servers.
+    Table containing the server daemon executable name as the first element and
+    optional additional commandline arguments as subsequent elements.
```

Comment:
thanks for documenting this. i think the readme.md changes need to go in https://github.com/neovim/nvim-lspconfig/blob/4e73c29aaf03451d5efcdf6c4747bac7c0dc1b00/scripts/README_template.md .

readme.md is auto-generated from it.

(future: we could avoid this confusion perhaps by generating to a separate file and linking to it from readme.md ...)